daemon/integration-tests/src/actors.rs

```diff
@@ -159,15 +165,29 @@ fn rand_usize_inclusive(start: usize, end: usize) -> usize {
 }
 
 impl Neovim {
-    pub async fn new_ethersync_enabled(initial_content: &str) -> (Self, PathBuf) {
+    // The caller should store the TempDir, so that it is not garbage collected.
```

Comment:
It's maybe a weird place, but what if we store it in the `Neovim` struct? Would it then be garbage collected at the correct time?

Reply:
I think it would be! It'd need to be an `Option`, because we can also construct Neovims without a TempDir inside it (with `new()`). It *does* feel a bit weird to put it there. Maybe I'd prefer it like it is?

Reply:
Seems fine. Until we fall into the trap to forget it :-D but I think it will just fail fast, so should be not a mean trap, but an easy one to get out of.